### PR TITLE
Fixing Alarm TimePicker

### PR DESCRIPTION
**Pull request by FunFrog-BY**
The Alarm TimePicker text field was only 256px wide, i don't know about
other languages, but in Russian caption just didn't fit. Stretched it to
the window width, now it's looking nice.

### CODE REVIEW

**Comment by jostrander** [comment]
Screen shot before/after?

**Comment by FunFrog-BY** [comment]
after

**Comment by FunFrog-BY** [comment]
Before:
before

File: src/renderer/ui/components/modals/AlarmModal.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ class AlarmModal extends Component {
       >
         <TimePicker
           hintText={TranslationProvider.query('button-text-click-here-to-set-alarm')}
+          textFieldStyle={{width: '100%'}}
```
Comment (MarshallOfSound) [inline src/renderer/ui/components/modals/AlarmModal.js]: http://www.material-ui.com/#/components/text-field

This should be replaced with the `fullWidth` property

Comment (FunFrog-BY) [inline src/renderer/ui/components/modals/AlarmModal.js]: Sorry, it`s my very first time with github, node, npm and of course material-ui. Can i make such minor cosmetic pull requests in future?

**Pull request merged**